### PR TITLE
Feature/coin-converter-CRW-52

### PR DESCRIPTION
**Pull request by AbdallaEmadEldinAbdalla**
[CRW-52](https://techxarya.atlassian.net/browse/CRW-52)

[CRW-52]: https://techxarya.atlassian.net/browse/CRW-52?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/pages/asset/index.tsx [hunk 1]
```diff
@@ -27,7 +29,14 @@ const AssetPage = () => {
...
     <Layout>
-      <Asset />
+      <Col className="gap-14">
+        <Asset />
+        <CoinProfitCalculator />
+        {asset?.id && <CoinConverter
+          preDefined
+          staticCoin={asset}
+        />}
+      </Col>
```
Comment (almujtabamohammed) [inline src/pages/asset/index.tsx:39]: CoinProfitCalculator should be inside Asset component.

File: src/components/shared/coinConverter/index.tsx [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+                                        <LoadingSpinner />
+                                    </Col>
+                                    :
+                                    <h3 className="text-center text-base font-bold text-white mb-4">{t('emptySearch')}</h3>)}
+                        </Col>
+                    </DropdownMenu.Content>
+                </DropdownMenu.Portal>
+            </DropdownMenu.Root>
+        );
+    }, [coins, dropdownItem, keyword, loading, onChangeKeyword, t]);
```
Comment (almujtabamohammed) [inline src/components/shared/coinConverter/index.tsx:132]: Please extract assets dropdown to a component, and match the design

File: src/components/shared/coinConverter/index.tsx [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+                <Col className="flex-[0.8] justify-center gap-6">
+                    <Row className="gap-4 items-center">
+                        {dropdown(firstCoin?.name || "Coins", props?.preDefined, 'first')}
+                        {numericInput(firstCoinAmount, onChnageFirstCoinAmount, !firstCoin?.name || !secondCoin?.name)}
+                    </Row>
+                    <Row className="relative">
+                        <Row className="w-full h-1 bg-black-1" />
+                        <Button className="absolute right-0 bottom-[-22px]" disabled>
+                            <ConvertButtonIcon />
+                        </Button>
```
Comment (almujtabamohammed) [inline src/components/shared/coinConverter/index.tsx:202]: This should switch assets?

File: src/components/shared/coinConverter/index.tsx [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
+                    </Row>
+                    <Row className="gap-4 items-center">
+                        {dropdown(secondCoin?.name || "Coins", false, 'second')}
```
Comment (almujtabamohammed) [inline src/components/shared/coinConverter/index.tsx:205]: Translate titles

**Review by almujtabamohammed — CHANGES_REQUESTED** [review]

File: src/components/shared/coinConverter/index.tsx (hunk 3 above)
Comment (AbdallaEmadEldinAbdalla) [reply src/components/shared/coinConverter/index.tsx:202]: no it's just icon for now

**Pull request merged**